setup.py

```diff
@@ -79,13 +79,17 @@ def get_all_extras() -> Dict:
         "connexion[swagger-ui]",
         *cli_deps
     ]
+    image_deps = [
```

Comment:
@dishmop  I think these should be added as dependencies of the skill rather than the package. What do you think @marcofavorito?